docs/howto/retry.rst

```diff
@@ -62,9 +71,12 @@ Implementing your own strategy
         min = 1
         max = 10
 
-        def get_schedule_in(self, attempts: int) -> int:
+        def get_schedule_in(self, exception:Exception, attempts: int, **kwargs) -> int:
             if attempts >= max_attempts:
                 return None
 
             return random.uniform(self.min, self.max)
 
+
+It's interesting to add a catch-all parameter ``**kwargs`` to make your strategy more
+resilient to possible evolutions of Procrastinate in the future.
```

Comment:
Even though we're changing the `*args` part of the method here :> They'll never be safe!

Reply:
Maybe adding a `*,` would be worth it, then...